src/frontends/paddle/tests/op_extension.cpp

```diff
@@ -0,0 +1,181 @@
+                                                      std::vector<std::string>{"Out"}),
+        std::make_shared<ov::frontend::OpExtension<>>("ov_CustomRelu_10",
+                                                      "fw_CustomRelu_10",
+                                                      std::vector<std::string>{"X"},
+                                                      std::vector<std::string>{"Out"}),
+        std::make_shared<ov::frontend::OpExtension<>>(
+            "ov_CustomRelu_11",
+            "fw_CustomRelu_11",
+            std::vector<std::string>{"X"},
+            std::vector<std::string>{"Out"},
+            std::map<std::string, std::string>{{"ov_attribute_1", "fw_attribute_1"},
+                                               {"ov_attribute_2", "fw_attribute_2"}}),
+        std::make_shared<ov::frontend::OpExtension<>>(
+            "ov_CustomRelu_12",
+            "fw_CustomRelu_12",
+            std::vector<std::string>{"X"},
+            std::vector<std::string>{"Out"},
+            std::map<std::string, std::string>{{"ov_attribute_1", "fw_attribute_1"},
+                                               {"ov_attribute_2", "fw_attribute_2"}},
+            std::map<std::string, ov::Any>{
+                {"ov_attribute_str", "string"},
+                {"ov_attribute_int", 4},
+                {"ov_attribute_bool", true},
+                {"ov_attribute_float", 4.f},
+                {"ov_attribute_vec_string", std::vector<std::string>{"str1", "str2", "str3"}},
+                {"ov_attribute_vec_int", std::vector<int>{1, 2, 3, 4, 5, 6, 7}},
+                {"ov_attribute_vec_bool", std::vector<bool>{true, false, true}},
+                {"ov_attribute_vec_float", std::vector<float>{1., 2., 3., 4., 5., 6., 7.}},
+            })};
+    return res;
+}
+
+INSTANTIATE_TEST_SUITE_P(PaddleOpExtensionTestViaUserClass,
+                         FrontEndOpExtensionTest,
+                         ::testing::Values(getTestDataOpExtensionViaUserClass()),
+                         FrontEndOpExtensionTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(PaddleOpExtensionViaPaddleConstructor,
+                         FrontEndOpExtensionTest,
+                         ::testing::Values(getTestDataOpExtensionViaPaddleConstructor()),
+                         FrontEndOpExtensionTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(PaddleOpExtensionViaCommonConstructor,
+                         FrontEndOpExtensionTest,
+                         ::testing::Values(getTestDataOpExtensionViaCommonConstructor()),
+                         FrontEndOpExtensionTest::getTestCaseName);
+
+TEST(FrontEndOpExtensionTest, paddle_opextension_relu) {
+    ov::Core core;
+    const auto extensions = std::vector<std::shared_ptr<ov::Extension>>{std::make_shared<ov::OpExtension<Relu1>>()};
+    core.add_extension(extensions);
+    std::string m_modelsPath = std::string(TEST_PADDLE_MODELS_DIRNAME);
+    std::string m_modelName = "relu/relu.pdmodel";
```

Comment:
we need to create new pd models with CustomRelu ops to really check the functionality. The tests check only extension registration, not functionality.
We can do it in the next PR